### PR TITLE
Nerfs Botany Maximum Yield + Plant Bags

### PR DESCRIPTION
**Pull request by ItsMarmite**
## What Does This PR Do
This PR lowers plant bag capacity from 100 to 40.

This PR also lowers the upper limit of the amount of plants botany can gain per tray per harvest to make it a little less ridiculous. The upper clamp that handles maximum plant yield has been lowered from 10 to 5, making it so Botany will at most be harvesting plants per yield in the 10s, rather than in the 20s.

## Why It's Good For The Game
The sheer amount of instantdeath plants botany can carry on their person is a little silly at times. Combining this with general plantspam from both war crime instantdeath plants, but also the amount of plants that can be grown, left on the floor to create alt+click piles of instant lag.

Basically, its a bit much and a bit silly. This PR should hopefully make the numbers just that little more reasonable.

## Testing
Grew a lot of tobacco, just not as much as usual. Put it in bags.

## Changelog
:cl:
tweak: Botany plant bags and associated subtypes now their maximum capacity lowered from 100 to 40.
tweak: Overall botany plant yield has been lowered.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/bags.dm [hunk 1]
```diff
@@ -202,8 +202,8 @@
 	name = "plant bag"
 	icon = 'icons/obj/hydroponics/equipment.dmi'
 	icon_state = "plantbag"
-	storage_slots = 100 //the number of plant pieces it can carry.
-	max_combined_w_class = 100 //Doesn't matter what this is, so long as it's more or equal to storage_slots * plants.w_class
+	storage_slots = 40 //the number of plant pieces it can carry.
+	max_combined_w_class = 40  //Doesn't matter what this is, so long as it's more or equal to storage_slots * plants.w_class
```
Comment (DGamerL) [inline code/game/objects/items/weapons/storage/bags.dm:206]:
```suggestion
	max_combined_w_class = 40 //Doesn't matter what this is, so long as it's more or equal to storage_slots * plants.w_class
```

**Review by DGamerL — APPROVED** [review]
Code looks good

**Review by Henri215 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]
